src/parser/jobs/rpr/modules/Procs.tsx

```diff
@@ -19,13 +19,15 @@ const SEVERITIES = {
 }
 
 const LOST_PROC_POTENCY = 60
+const ENCHANCED_GIBBET_IGNORE_INTERVAL = 23000
 
 export class Procs extends CoreProcs {
 	static override handle = 'enhanced procs'
 	static override title = t('rpr.procs.title')`Enhanced Procs`
-	static override debug = false
+	static override debug = true
```

Comment:
Don't commit debug = true
```suggestion
	static override debug = false
```